### PR TITLE
Add `observationType` and `hasQualityMeasurement`

### PR DESCRIPTION
**Pull request by dr-shorthair**
Additional Observation properties mentioned in OMS

### CODE REVIEW

**Review by KathiSchleidt — APPROVED** [review]
Looks good!

**Review by maximelefrancois86 — CHANGES_REQUESTED** [review]
Why is `observationType` in the SOSA namespace ? This object property is not defined in sosa.ttl  and not documented in the specification.

**Comment by dr-shorthair** [comment]
It comes from OMS. 
I put it in `sosa:` ns in anticipation of it being added to sosa.ttl. 
See https://github.com/w3c/sdw-sosa-ssn/issues/132

**Comment by dr-shorthair** [comment]
My suggestion is that we put all the new things in `sosa:` provisionally. 
We can then consider if we need more horizontal modules in due course, which might involve moving some terms into new namespaces. 

(We have a lot of branches and need to do some merging soon else it'll get horribly tangled.)

**Comment by dr-shorthair** [comment]
Closes #124

File: ssn/integrated/sosa-oms.ttl [hunk 1]
```diff
@@ -307,6 +300,12 @@ sosa:Observation ogc:implements <http://www.opengis.net/spec/om/3.0/req/obs-cpt/
     schema:rangeIncludes sosa:Observation ;
     rdfs:isDefinedBy sosa-oms: .
 
+  dqv:hasQualityMeasurement
+    ogc:implements <http://www.opengis.net/spec/om/3.0/req/obs-core/AbstractObservationCharacteristics/resultQuality-sem> ;
+    skos:definition "Information pertaining to the data quality of the result." ;
+    schema:domainIncludes osa:Observation ;
```
Comment (sgrellet) [inline ssn/integrated/sosa-oms.ttl]: sosa:Observation instead of osa:Observation

I guess we'll discuss the range afterwards

Comment (sgrellet) [reply ssn/integrated/sosa-oms.ttl]: the rest looks ok to me

**Review by sgrellet — APPROVED** [review]
fine with me

**Comment by dr-shorthair** [comment]
> Why is `observationType` in the SOSA namespace ? This object property is not defined in sosa.ttl and not documented in the specification.

For clarity at this stage, I've moved all the terms that are defined in the SOSA-OMS graph into the `sosa-oms:` ns. 
We can consider the final destination in #132. 
@maximelefrancois86 can you approve now so we can merge?

**Review by maximelefrancois86 — APPROVED** [review]

**Pull request merged**